code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -648,6 +648,31 @@
 	muzzled_noises = list("annoyed")
 	emote_type = EMOTE_SOUND | EMOTE_MOUTH
 
+/datum/emote/living/carbon/human/hiss
+	key = "hiss"
```

Comment:
like above, this is gonna need a different key I think

Reply:
seeing as i cannot come up with a good alternative, i hope just changing one letter is sufficient!